### DISCUSSION TITLE
My registrar says Github Ips are Blacklisted

### DISCUSSION THREAD

**Discussion by Abiel0** [community/community#143296] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

Today I added my domains to domain registrar. But it says The Ip is in a blacklist. I checked a blacklist checker and it said this blacklist is made by spamhaus and to check the reason https://check.spamhaus.org/sbl/query/SBL658687 . I send them a mail explaining These github ips are safe , How can we resolve this problem so I can add my domains to my registrar.

**Comment by Abiel0** [comment]
Because I dk

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Abiel0 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by LiteBrite82** [comment]
Hi @Abiel0 
The pages team is on it. See also: https://github.com/orgs/community/discussions/143506
